### PR TITLE
Read more information from the input file

### PR DESCRIPTION
**Pull request by Anchewei**
- Read Npar and theta from the input file
- Change the precision in the input and output file (the errors can be further reduced!)
- Add some error messenger in CreatePar.py

### CODE REVIEW

**Comment by Anchewei** [comment]
- Free memory

File: clean.sh
Comment (willsonho2000) [inline clean.sh:1]: You can add this line into Makefile: make clean

**Review by willsonho2000 — CHANGES_REQUESTED** [review]
Use make clean instead of clean.sh?

File: clean.sh
Comment (Anchewei) [reply clean.sh:1]: clean.sh is only used to remove the input/output files.

**Review by willsonho2000 — APPROVED** [review]
Ok, good to go.

**Pull request merged**